leasing/serializers/utils.py

```diff
@@ -305,3 +306,34 @@ def get_file_url(self, obj):
 
     def get_file_filename(self, obj):
         return os.path.basename(obj.file.name)
+
+
+def validate_seasonal_day_for_month(day: int | None, month: int | None):
+    if day is None and month is None:
+        return
+
+    if (day is None and month is not None) or (day is not None and month is None):
```

Comment:
Could it be `if day is None or month is None:`? It is a bit simpler.

Reply:
Correct, I've added validation for both cases now though. So if one is None and other is not